compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SemanticAnalyzer.java

```diff
@@ -1687,11 +1687,19 @@ private void validateVariableDefinition(BLangExpression expr) {
         // var a = [ x, y, ... ];
         // var a = { x : y };
         // var a = new ;
+        // var a = ( 1, 2, .. );
         final NodeKind kind = expr.getKind();
         if (kind == NodeKind.RECORD_LITERAL_EXPR || kind == NodeKind.ARRAY_LITERAL_EXPR
                 || (kind == NodeKind.Type_INIT_EXPR && ((BLangTypeInit) expr).userDefinedType == null)) {
             dlog.error(expr.pos, DiagnosticCode.INVALID_ANY_VAR_DEF);
         }
+        if (kind == NodeKind.BRACED_TUPLE_EXPR) {
+            BLangBracedOrTupleExpr bracedOrTupleExpr = (BLangBracedOrTupleExpr) expr;
+            if (bracedOrTupleExpr.expressions.size() > 1 &&
+                    bracedOrTupleExpr.expressions.stream().anyMatch(literal -> literal.getKind() == NodeKind.LITERAL)) {
+                dlog.error(expr.pos, DiagnosticCode.INVALID_ANY_VAR_DEF);
```

Comment:
we should check the status of the validateVariableDefinition in the caller method and set it's typed to error.